doc/nrf/releases/release-notes-latest.rst

```diff
@@ -407,6 +423,7 @@ The following list summarizes the most important changes inherited from upstream
     * Deprecated legacy TCP stack (TCP1).
     * Added multiple minor TCP2 bugfixes and improvements.
     * Added network management events for DHCPv4.
+    * Fixed getaddrinfo to actually respect socket type hints.
```

Comment:
Is this covered by https://github.com/nrfconnect/sdk-nrf/pull/3647/files line 484?